packages/devtools_app/lib/src/shared/ui/filter.dart

```diff
@@ -193,8 +193,7 @@ mixin FilterControllerMixin<T> on DisposableController
 /// the filter state.
 class FilterDialog<T> extends StatefulWidget {
   FilterDialog({super.key, required this.controller})
-    : assert(controller.queryFilterArgs.isNotEmpty),
```

Comment:
This was throwing an assert error, and as far as I can tell we are not expecting the `queryFiltersArgs` to be non-empty anywhere in this widget.